app/views/layouts/application.html.erb

```diff
@@ -31,11 +35,20 @@
           <strong class="govuk-tag govuk-phase-banner__content__tag">
             beta
           </strong>
-          <span class="govuk-phase-banner__text">
-            This is a new service – your
-            <%= mail_to contact_details[:feedback_email], "feedback", class: "govuk-link"  %>
-            will help us to improve it.
-          </span>
+
+          <% if ENV["FEEDBACK_FISH_ID"].present? %>
+            <span class="govuk-phase-banner__text">
+              This is a new service – your
+              <a class="govuk-link" href="#"
+                data-feedback-fish
+                data-feedback-fish-userid="<%= @planning_application["agent_email"] if @planning_application %>"
```

Comment:
Since there is no authenticated user we can send the agent's email here instead if there is one.

Since feedback fish will capture the url anyway we can identify the relevant planning application in case an officer needs to reach out